packages/enzyme-adapter-react-14/src/ReactFourteenAdapter.js

```diff
@@ -136,6 +142,20 @@ class ReactFourteenAdapter extends EnzymeAdapter {
       batchedUpdates(fn) {
         return ReactDOM.unstable_batchedUpdates(fn);
       },
+      getWrappingComponentRenderer() {
+        return {
+          ...this,
+          getNode() {
+            return instance ? instanceToTree(instance._reactInternalInstance).rendered : null;
+          },
+          render(el, context, callback) {
+            if (!instance) {
+              throw new Error('The wrapping component may not be updated if the root is unmounted.');
+            }
+            instance.setWrappingComponentProps(el.props, callback);
+          },
+        };
+      },
```

Comment:
@ljharb This seemed a bit trivial to dedupe into `enzyme-adapter-utils`. Do you agree?

Reply:
if the `getNode` logic is the same, and is unlikely to differ (or can be passed in via callback) then it seems worth deduping as much as possible.

Reply:
Extracted!